src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -342,6 +342,17 @@ protected final String getSuccessfulDestination() {
         return triggers.toMap();
     }
 
+    public List<TriggerDescriptor> getTriggerDescriptors() {
+        List<TriggerDescriptor> result = new ArrayList<TriggerDescriptor>();
+        for (TriggerDescriptor d: Trigger.for_(this)) {
+            if (d instanceof TimerTrigger.DescriptorImpl) {
+                continue;
```

Comment:
Not to `for_`, since that operates without context. Rather you would have to call `h.filterDescriptors` when invoking `descriptorList`. Or use a higher-level control which does this automatically, if there is one.